demo/js/demo.js

```diff
@@ -110,6 +111,25 @@ function loadRomFromURL(URL) {
     });
 }
 
+function addClickListeners() {
+    for(key in KEY_DOMKEY){
+        KEY_DOMKEY[key].mousedown({keyCode: key}, function(e){
+            if(e.data.keyCode in KEY_DOMKEY){
```

Comment:
Thanks for the contribution!

One thing I noticed:
Given that you are creating a new callback function for each key, couldn't you just call the interpreter with the key index directly?